### DISCUSSION TITLE
EMQX Replica nodes are not created

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx-operator#500] [Q&A] [answered]
HI,
When I apply the yamll https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v2alpha1/emqx-full.yaml the core nodes are created but replica nodes are not. How to solve this?
Thank you in advance..

**Comment by Rory-Z** [comment]
Hi, @alexa22013 EMQX Operator will wait EMQX core node ready before create EMQX replicant node, could you please make sure the EMQX core node status, maybe you can check EMQX resource status and event, and EMQX Operator logs

**Comment by alexa22013** [comment]
Hey thanks for the reply. I checked the core node status, its running fine for last 8 hours. Also checked operator events no specific issues found in it. The thing is it was working fine with replicant nodes few days before, I just deleted the deployment to change some configs and after that this issue came in

**Reply by Rory-Z** [reply]
Sorry, my english is not good, I am a little confused, the issue is still exist ?

**Reply by alexa22013** [reply]
yes the issue still exists..

**Reply by alexa22013** [reply]
<img width="1440" alt="Screenshot 2022-12-05 at 6 41 35 PM" src="https://user-images.githubusercontent.com/119624630/205645889-f91aec6e-ea8b-4e20-8336-037e536047e2.png">

**Reply by Rory-Z** [reply]
> <img alt="Screenshot 2022-12-05 at 6 41 35 PM" width="1440" src="https://user-images.githubusercontent.com/119624630/205645889-f91aec6e-ea8b-4e20-8336-037e536047e2.png">

The content of the event looks like the pod was never ready, could you please show EMQX pods and EMQX resource status for me

**Reply by alexa22013** [reply]
Emqx pod status
<img width="1440" alt="Screenshot 2022-12-07 at 1 00 23 AM" src="https://user-images.githubusercontent.com/119624630/206005114-dbdfca7b-3983-443a-9174-9c2273ccc2ca.png">

emqx resource status
<img width="1440" alt="Screenshot 2022-12-07 at 1 03 46 AM" src="https://user-images.githubusercontent.com/119624630/206005330-05f71620-45d2-46ac-a4f9-c9daca5312b2.png">

**Reply by Rory-Z** [reply]
Sorry for delay, could you please describe your steps in as much detail as possible so that I can reproduce your question?

**Reply by alexa22013** [reply]
So when I deploy the emqx resource for the first time it works fine, both core and replicant nodes are created. But when I delete the resource to change the bootstrapconfig and redeploy the emqx resource, the replicant nodes are not created

But what I noticed is that when I delete the resource along with the persistent volume claims that is use by the resource pods and I redeploy, everthing works fine.

**Reply by Rory-Z** [reply]
Did you modify the number of EMQX Core nodes when you redeploy EMQX ?  If so, perhaps this is the reason

**Reply by alexa22013** [reply]
No, the number of core nodes were same

**Reply by Rory-Z** [reply]
> But when I delete the resource to change the bootstrapconfig and redeploy the emqx resource, the replicant nodes are not created

could you please show me exactly what you have changed?

**Reply by webfrank** [reply]
Hi, I got the same behavior, if I need to change the bootstrapConfig I need to delete and re-create the cluster but replicant nodes are not created anymore. I need to delete the persistent volume (losing all the configs).

**Reply by Rory-Z** [reply]
Hi, @webfrank Cloud you please create an issue describing your steps in detail? This will help us a lot to reproduce and solve the problem

**Reply by Rory-Z** [reply]
I see this issue: https://github.com/emqx/emqx/issues/9661

**Comment by Rory-Z** [comment, marked as answer]
Please update EMQX version to 5.0.14, and delete `bootstrap_user` in EMQX Dashboard System Users, and EMQX Cluster will work

**Reply by alexa22013** [reply]
How to delete the bootstrap_user in yaml config?

**Reply by alexa22013** [reply]
Is there anyway to avoid this issue without delete bootstrap user from dashboard?
